index.js

```diff
@@ -180,72 +180,89 @@
   //  Value :: Location
   var Value = 'Value';
 
-  //  _funcPath :: (Boolean, Array String, a) -> Nullable Function
-  function _funcPath(allowInheritedProps, path, _x) {
-    var x = _x;
-    for (var idx = 0; idx < path.length; idx += 1) {
-      var k = path[idx];
-      if (x == null || !(allowInheritedProps || has(k, x))) return null;
-      x = x[k];
-    }
-    return typeof x === 'function' ? x : null;
-  }
+  //  Prefix :: String
+  var Prefix = 'fantasy-land/';
 
   //  funcPath :: (Array String, a) -> Nullable Function
-  function funcPath(path, x) {
-    return _funcPath(true, path, x);
+  function funcPath(path, value) {
+    var x = value, len = path.length;
```

Comment:
I'm fairly certain `length` caching is unnecessary in general. Modern browsers do it themselves.